securedrop_client/gui/widgets.py

```diff
@@ -966,15 +966,33 @@ def get_current_source(self):
         if source_widget and source_exists(self.controller.session, source_widget.source_uuid):
             return source_widget.source
 
-    def set_snippet(self, source_uuid, message_uuid, content):
-        """
-        Given a UUID of a source, if the referenced message is the latest
-        message, then update the source's preview snippet to the referenced
-        content.
-        """
-        source_widget = self.source_widgets.get(source_uuid)
+    def get_source_widget(self, source_uuid: str) -> QListWidget:
```

Comment:
Should the return type hint be `Optional[QListWidget]`?

Reply:
good point, will update